### DISCUSSION TITLE
HB-RF-USB-2 mit HM-MOD-RPI-PCB wird nicht erkannt

### DISCUSSION THREAD

**Discussion by itn3rd77** [OpenCCU/OpenCCU#1677] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,

ich wollte gerne meinen "alten" Raspberry Pi 3 durch eine OVA Installation auf meiner Synology mit HB-RF-USB-2 ablösen.

Also RaspberryMatic (3.61.7.20220115) OVA image auf der Synology installiert und das HM-MOD-RPI-PCB von dem Raspberry Pi 3 auf die HB-RF-USB-2 gesteckt und per USB an die Synology angeschlossen.

Leider wird kein Funkmodul erkannt. In der Ausgabe vom dmesg wird zwar das HB-RF-USB-2 erkannt, aber das `Could not verify device signature` sieht denke ich nicht gut aus, oder?

```
[ 3496.695304] usb 3-1: Found HB-RF-USB-2 with serial b65093db4cc1e8119e376b713b0549ec at usb-0000:00:18.0-1
[ 3530.231151] usb 3-1: USB disconnect, device number 5
[ 3556.689127] usb 3-1: new full-speed USB device number 6 using uhci_hcd
[ 3557.043051] usb 3-1: New USB device found, idVendor=10c4, idProduct=8c07, bcdDevice= 1.01
[ 3557.043060] usb 3-1: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[ 3557.043064] usb 3-1: Product: HB-RF-USB-2
[ 3557.043067] usb 3-1: Manufacturer: Alexander Reinert
[ 3557.043070] usb 3-1: SerialNumber: b65093db4cc1e8119e376b713b0549ec
[ 3557.051228] usb 3-1: DKey: 6264333930353662 3131386531636334 3137623637336539 6365393435306233
[ 3557.051238] usb 3-1: HKey: 19eee21d6f361bf5 658b2aa7815cec81 4957cb3887056d7c f736953f6f821ee5
[ 3557.051241] usb 3-1: SKey: 0000000000000000 0000000000000000 0000000000000000 0000000000000000
[ 3557.051244] usb 3-1: SKey: 0000000000000000 0000000000000000 0000000000000000 0000000000000000
[ 3557.051248] usb 3-1: PKey: f50048a8d7cc3f44 ae8460d13cf61d60 2dc161f5272a5876 88c996b27737fbae
[ 3557.051251] usb 3-1: PKey: dd24b0aced521e88 cc457acb4b7c49a4 6745032ab2079847 becfb9f3af25d41f
[ 3557.051259] usb 3-1: Could not verify device signature
[ 3557.051264] usb 3-1: Found HB-RF-USB-2 with serial b65093db4cc1e8119e376b713b0549ec at usb-0000:00:18.0-1
```

Das Modul HM-MOD-RPI-PCB hat die aktuellste Firmware (2.8.6) und so wie es aussieht haben das ja auch andere Leute in dieser Form im Einsatzt. Kann mir jemand mit der Fehlermeldung `Could not verify device signature` weiterhelfen? Hab ich evtl. einen Lötfehler oder ist das ein reines Software Problem?

Danke für eure Hilfe!

PS: Ja ich weiß das Modul HM-MOD-RPI-PCB ist veraltet, aber das neue Modul hat > 11 Wochen Lieferzeit und es wird also noch ein bischen dauern bis es ankommt. Würde aber gerne vorher schon mal weg von dem Raspberry Pi 3 .

**Comment by jens-maus** [comment, marked as answer]
> Leider wird kein Funkmodul erkannt. In der Ausgabe vom dmesg wird zwar das HB-RF-USB-2 erkannt, aber das `Could not verify device signature` sieht denke ich nicht gut aus, oder?

Nein, das ist keine Fehlermeldung (ERROR) sondern lediglich eine notice darüber das die Signatur der `HB-RF-USB-2` Platine nicht überprüft werden konnte. Das (die Signaturprüfung) ist etwas an dem @alexreinert schon länger arbeitet um Mockups seiner Platinen in Zukunft unterbinden zu können. Jedoch wird diese Signaturprüfung bis jetzt noch nicht hart umgesetzt bzw. es würde eine neuere Firmware des `HB-RF-USB-2` notwendig machen.

Insofern kann diese Meldung ignoriert werden und sollte auch nicht dafür verantwortlich sein, das dein `HM-MOD-RPI-PCB` beim Hochfahren von RaspberryMatic anscheinend nicht erkannt wird. Du solltest also nach anderen Ursachen einmal suchen und dir einmal genauer anschauen was beim Hochfahren so auf der virtuellen Konsole ausgegeben wird.

**Reply by itn3rd77** [reply]
Danke für die schnelle Antwort und Klarstellung Jens!
